packages/skeleton-react/src/components/Progress/Progress.tsx

```diff
@@ -0,0 +1,68 @@
+import { useEffect } from "react";
+import { ProgressProps } from "./types";
+
+export default function Progress({
+	value,
```

Comment:
I might suggest considering appending an optional `id` prop. This could be useful your console error to identify *which* component is erroring. Apply it to the parent element.

Reply:
This feels like overkill since svelte already shows where you are using the component (file and linenumber)